CONTRIBUTING.md

```diff
@@ -1,23 +1,23 @@
 Thank you for considering contributing to Dynamic Config!
 Please take a moment to review this document to make the contribution process easy and effective.
 
-## Reporting a bug
+<span style=" font-size:2.5em;">Reporting a bug 🐞</span>
```

Comment:
This was a test for setting font size in markdown that I totally forgot to omit, but it's here now so 😅